Source/ASTextNode.mm

```diff
@@ -130,6 +130,23 @@ - (BOOL)isEqual:(ASTextNodeRendererKey *)object
   return renderer;
 }
 
+static AS::RecursiveMutex __sharedRendererCacheInstanceLock__;
```

Comment:
nit: should this go at the top with the other locks? Or is the thinking to clean this up if it works?

Reply:
Which other locks? Since these are static C functions I put the lock right next to them. Is there a better place to put them?

Reply:
Ignore me, I was looking at a different file 😅